src/Map.js

```diff
@@ -116,4 +121,23 @@ const ViewChanger = ({center}) => {
     return null;
 }
 
+const  Legend = ({metadata}) => {
+    const map = useMap()
+    useEffect(() => {
+      if (map && metadata.updated_at) {
```

Comment:
We want to make sure something is actually inside of the metadata object before we go through with adding the legend to the map, otherwise it'll populate prematurely without waiting for the actual data.